scopes/lanes/ui/hooks/lanes-query/lanes-query.ts

```diff
@@ -24,13 +24,16 @@ const GET_LANES = gql`
   }
 `;
 
-export function useLanesQuery(): { lanes?: LaneModel[] } & Omit<QueryResult<LanesQueryResult>, 'data'> {
+export function useLanesQuery(): { lanes?: LaneModel[]; checkedoutLane?: string } & Omit<
+  QueryResult<LanesQueryResult>,
+  'data'
+> {
   const { data, ...rest } = useDataQuery<LanesQueryResult>(GET_LANES);
   const { scope, loading } = useScopeQuery();
-  const lanes = data && LanesModel.from(data, scope);
   return {
     ...rest,
     loading: rest.loading || !!loading,
-    lanes,
+    lanes: data && LanesModel.from(data, scope),
+    checkedoutLane: data?.lanes?.getCurrentLaneName,
```

Comment:
that's.. really confusing. you are checking if checkoutLane !=== current lane in a different file

Reply:
From the CLI's perspective (`getCurrentLane`) exposed from the endpoint represents the currently checked out lane
which is always `main` on a scope but locally can be whatever the lane you are currently switched to

In the UI, the user can view a lane without having to switch to it. 

When you are switched out locally to the lane you are currently viewing on the UI,
the command to update your components from remote is different 
v/s
when you are just viewing a lane you are not switched to